src/rez/package_maker__.py

```diff
@@ -31,7 +31,8 @@
         Or(basestring, [basestring]),
         {
             "command": Or(basestring, [basestring]),
-            Optional("requires"): [package_request_schema]
+            Optional("requires"): [package_request_schema],
+            Optional("run_in_root"): bool
```

Comment:
`run_in_root` is much too general a setting name for this case. I'd suggest `tests_artifact_path` or similar.